prime-angular-frontend/src/app/modules/adjudication/pages/health-authorities/health-auth-care-types-page/health-auth-care-types-page.component.ts

```diff
@@ -58,14 +61,20 @@ export class HealthAuthCareTypesPageComponent implements OnInit {
     }
   }
 
-  public addCareType(careType: string = '') {
+  public addCareType(careType: HealthAuthorityCareTypeConfig = null) {
     this.careTypes.push(this.fb.group({
-      careType: [careType ?? '', Validators.required]
+      careType: [careType ?? null, Validators.required]
     }));
   }
 
   public removeCareType(index: number) {
-    this.careTypes.removeAt(index);
+    const careTypeId = this.careTypes.value[index].careType.id;
+    this.healthAuthResource.getHealthAuthorityCareTypeSiteIds(this.route.snapshot.params.haid, careTypeId)
+      .subscribe((healthAuthoritySites) => {
+        (!healthAuthoritySites.length)
+          ? this.careTypes.removeAt(index)
+          : this.toastService.openErrorToast('Health authority care type could not be deleted. One or more sites are using it');
```

Comment:
To reduce the amount of text the user has to read in a toast could simplify to
``` Care type could not be removed, one or more sites are using it```

Reply:
Simplified